docs/automations/integrations/github-actions/dispatch-github-action/README.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Automation - Dispatch GitHub Actions
+description: Automatically trigger GitHub Actions based on PR content like changed resources, source or target branch, slash commands, and more.
+---
+# Dispatch GitHub Actions
+
+
+<!-- --8<-- [start:example]-->
+Automatically trigger GitHub Actions based on PR content like changed resources, source or target branch, slash commands, and more.
+
+![Automatically Dispatch GitHub Actions](/automations/integrations/github-actions/dispatch-github-action/dispatch-github-action.png)
+
+=== "By Branch"
+    !!! info "Configuration Description"
+        Conditions (all must be true):
+
+        * The PR source or target branch matches a specified format.
+
+        Automation Actions:
+
+        * Trigger a manual dispatch for the specified CI pipeline.
+
+    <div class="automationExample" markdown="1">
+    !!! example "Dispatch GitHub Actions by Branch"
+        ```yaml+jinja
+        --8<-- "docs/downloads/automation-library/integrations/github-actions/dispatch-github-action/dispatch_github_action_branch.cm"
+        ```
+        <div class="result" markdown>
+        <span>
+        [:octicons-download-24: Download this example as a CM file.](/downloads/automation-library/integrations/github-actions/dispatch-github-action/dispatch_github_action_branch.cm){ .md-button }
+        </span>
+        </div>
+    </div>
+=== "Using Labels"
+    !!! info "Configuration Description"
+        Conditions (all must be true):
+
+        * The PR has one or more specified labels applied to it.
+
+        Automation Actions:
+
+        * Trigger a manual dispatch for the specified CI pipeline.
+
+    <div class="automationExample" markdown="1">
+    !!! example "Dispatch GitHub Actions Using Labels"
```

Comment:
How is it invoked using labels? It looks like we add the label together with the action